### PR TITLE
doc and test app updates for Inbound changes

### PR DESCRIPTION
**Pull request by sbenedicadb**

### CODE REVIEW

File: AEPMessaging/Sources/Messaging+EdgeEvents.swift [hunk 1]
```diff
@@ -269,74 +275,66 @@ extension Messaging {
...
+        var decisioning: [String: Any] = [
             MessagingConstants.XDM.IAM.Key.PROPOSITION_EVENT_TYPE: propositionEventType,
             MessagingConstants.XDM.IAM.Key.PROPOSITIONS: propositions
         ]
+        
+        // only add `propositionAction` data if this is an interact event
+        if eventType == .inappInteract {
+            let propositionAction: [String: String] = [
+                MessagingConstants.XDM.IAM.Key.ID: interaction ?? "",
+                MessagingConstants.XDM.IAM.Key.LABEL: interaction ?? ""
```
Comment (rymorale) [inline AEPMessaging/Sources/Messaging+EdgeEvents.swift:311]: id and label should both contain the interaction?

Comment (sbenedicadb) [reply AEPMessaging/Sources/Messaging+EdgeEvents.swift:311]: for now, yes. 

in the future we may allow there to be a separation of these values (if it's supported in the authoring ui)

File: AEPMessaging/Sources/PropositionInfo.swift [hunk 2]
```diff
@@ -23,4 +23,11 @@ extension PropositionInfo {
     var correlationId: String {
```
Comment (rymorale) [inline AEPMessaging/Sources/PropositionInfo.swift:23]: nit: as this is no longer used can it be removed?

File: AEPMessaging/Tests/UnitTests/Messaging+EdgeEventsTests.swift [hunk 3]
```diff
@@ -459,16 +459,19 @@ class MessagingEdgeEventsTests: XCTestCase {
         let decisioningMap = experienceMap?["decisioning"] as? [String: Any]
         let propositionEventTypeMap = decisioningMap?["propositionEventType"] as? [String: Any]
         XCTAssertEqual(1, propositionEventTypeMap?["interact"] as? Int)
+        let propositionActionMap = decisioningMap?["propositionAction"] as? [String: Any]
+        XCTAssertEqual(2, propositionActionMap?.count)
+        XCTAssertEqual(mockInteraction, propositionActionMap?["id"] as? String)
+        XCTAssertEqual(mockInteraction, propositionActionMap?["label"] as? String)
```
Comment (rymorale) [inline AEPMessaging/Tests/UnitTests/Messaging+EdgeEventsTests.swift:465]: ah i see interaction should be in both id and label :)

**Review by rymorale — APPROVED** [review]
looks good to me 👍

**Pull request merged**